src/main/kotlin/com/cognifide/gradle/aem/common/NotifierFacade.kt

```diff
@@ -94,6 +96,20 @@ class NotifierFacade private constructor(private val aem: AemExtension) {
 
         const val DORKBOX_HIDE_AFTER_SECONDS = 5L
 
+        private const val DORKBOX_MAIN_TEXT_FONT = "Source Code Pro BOLD 13"
```

Comment:
what if this font will be missing?

Reply:
That's the default font from the library, just changing its size one point up.